src/lib/build.js

```diff
@@ -41,11 +43,20 @@ const getBuildOptions = ({ cachedConfig, options: { context, cwd, debug, dry, js
 
 /**
  * run the build command
- * @param {BuildConfig} options
+ * @param {BuildConfig} buildOptions
+ * @param {import('../commands/base-command').BaseCommand} command
+ * @param {import('commander').OptionValues} commandOptions
  * @returns
  */
-const runBuild = async (options) => {
+const runBuild = async (buildOptions, command, commandOptions) => {
```

Comment:
Conceptually, I feel like it might be best for this function to receive just the options it needs rather than receiving the entire command. Because this is used by multiple commands, it might not be obvious that the arguments will immediately change as soon as one of the commands change.

Not a blocker, though.